### PR TITLE
DM-11790 Move jointcal document out of jointcal and into technote

### PR DESCRIPTION
**Pull request by parejkoj**

### CODE REVIEW

**Comment by timj** [comment]
Your commit author emails are messed up so that GitHub doesn't know who you are.

File: DMTN-036.tex [hunk 1]
```diff
@@ -0,0 +1,808 @@
...
+models which are more precise than when fitted independently. This
+is especially true when the images are deeper than the astrometric
+reference catalogs. In the ``Astromatic'' software suite, this simultaneous
+astrometry functionality is fulfilled by ``SCAMP''. The code we
+describe here has similar aims, but follows a slightly different route.
+Jointcal is built on top of the the LSST Data Management software
+stack.}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% User specified LaTeX commands.
+\newcommand{\ClName}[1]{{\bf #1}}
```
Comment (timj) [inline DMTN-036.tex]: Please live in the 1990s in a world where latex2e exists.

File: DMTN-036.tex [hunk 2]
```diff
@@ -0,0 +1,808 @@
...
+\makeatletter
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LyX specific LaTeX commands.
+
+\title[jointcal]{jointcal: Simultaneous Astrometry \& Photometry for thousands of
+Exposures with Large CCD Mosaics}
+\author{John Parejko (University of Washington), Pierre Astier (LPNHE/IN2P3/CNRS
+Paris)}
+\setDocRef{DMTN-036}
+\date{\today}
```
Comment (timj) [inline DMTN-036.tex]: When you release and tag this, fix the date.

File: DMTN-036.tex [hunk 3]
```diff
@@ -0,0 +1,808 @@
...
+\setDocChangeRecord{%
+  \addtohist{1}{2017-09-01}{Unreleased.}{John Parejko}
+}
+
+\makeatother
+
+\usepackage{xunicode}
+\begin{document}
+\maketitle
+\tableofcontents{}
```
Comment (timj) [inline DMTN-036.tex]: Remove this line. You have a second empty table of contents in the PDF.

File: DMTN-036.tex [hunk 4]
```diff
@@ -0,0 +1,808 @@
...
+% Order: oldest first.
+% Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
+% See LPM-51 for version number policy.
+\setDocChangeRecord{%
+  \addtohist{1}{2017-09-01}{Unreleased.}{John Parejko}
+}
+
+\makeatother
+
+\usepackage{xunicode}
```
Comment (timj) [inline DMTN-036.tex:38]: This package is already loaded by the class.

File: DMTN-036.tex [hunk 5]
```diff
@@ -0,0 +1,808 @@
...
+proper motions (where applicable), and parameters describing the differential
+effect of atmospheric refraction on the position of objects. It is
+clear that one cannot fit simultaneously the position on the sky and
+the mappings from CCD coordinates to the sky, without extra constraints:
+the ``sky'' coordinate system is then undefined, and one needs reference
+positions in order to fully define this frame. We use the GAIA catalog
+as our reference catalog, and may supplement it with deeper data (e.g.
+PS1) where available.
+
+SCAMP \footnote{See \href{http://www.astromatic.net/software/scamp}{http://www.astromatic.net/software/scamp}}is
```
Comment (timj) [inline DMTN-036.tex]: Cite SCAMP (I really don't like footnote URL citations).

File: DMTN-036.tex [hunk 6]
```diff
@@ -0,0 +1,808 @@
...
+measurements with respect to all unknown parameters required to describe
+the data. These parameters are mostly in two sets: the position on
+the sky of the objects in common between the images, and the mapping
+of each image to the sky. To these obvious parameters, one can add
+proper motions (where applicable), and parameters describing the differential
+effect of atmospheric refraction on the position of objects. It is
+clear that one cannot fit simultaneously the position on the sky and
+the mappings from CCD coordinates to the sky, without extra constraints:
+the ``sky'' coordinate system is then undefined, and one needs reference
+positions in order to fully define this frame. We use the GAIA catalog
```
Comment (timj) [inline DMTN-036.tex:63]: I believe you are using the Gaia catalog not the GAIA catalog.

File: DMTN-036.tex [hunk 7]
```diff
@@ -0,0 +1,808 @@
...
+be reproduced in the context of our package. We provide residual ntuples
+and hope that the first serious users will contribute plotting tools.
+
+Loading input catalogs and their metadata from disk is a large fraction
+of the total time. We can save time by re-using the input catalog
+to fit a similar style of multi-component model for the relative and
+absolute photometry.
+
+The plan of this note is as follows: we first sketch the algorithm
+(\S\ref{sec:algo}). We provide our least-squares formulation in
```
Comment (timj) [inline DMTN-036.tex]: Consider `\secref`.

File: DMTN-036.tex [hunk 8]
```diff
@@ -0,0 +1,808 @@
...
+\S\ref{sec:mathematical-formalism}, and describe the how we evaluate
+the derivatives with respect to the parameters. We then describe how
+we associate the measurements of each object in different exposures
+in \S \ref{sec:catalog-association}.
```
Comment (timj) [inline DMTN-036.tex]: Should not be extra space here.

**Comment by parejkoj** [comment]
Deleting my previous comment: somehow my laptop .gitconfig lost its email address. I'll rebase again and push.

**Comment by parejkoj** [comment]
Although github still doesn't realize that my laptop and desktop are both me (they both have the same user email configured). Any ideas there are welcome.

File: DMTN-036.tex [hunk 9]
```diff
@@ -0,0 +1,808 @@
...
+is especially true when the images are deeper than the astrometric
+reference catalogs. In the ``Astromatic'' software suite, this simultaneous
+astrometry functionality is fulfilled by ``SCAMP''. The code we
+describe here has similar aims, but follows a slightly different route.
+Jointcal is built on top of the the LSST Data Management software
+stack.}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% User specified LaTeX commands.
+\newcommand{\ClName}[1]{{\bf #1}}
+\newcommand{\RoutineName}[1]{\texttt{#1}}
```
Comment (timj) [inline DMTN-036.tex]: There is `\code{}`

File: DMTN-036.tex [hunk 10]
```diff
@@ -0,0 +1,808 @@
...
+to serve as a reference with a fixed identity mapping. The distortion
+model \ClName{SimplePolyModel} allows that. 
+\item Chose identity projectors (the class \ClName{IdentityProjector}
+does precisely that). 
+\end{itemize}
+So, fitting transformations between image sets can be done with the
+provided code.
+
+\section{Run example}
+\begin{verbatim}
```
Comment (timj) [inline DMTN-036.tex]: Use the `{lstlisting}` environment here.

File: DMTN-036.tex [hunk 11]
```diff
@@ -0,0 +1,808 @@
...
+in the FITS header. This projection is often used to generate re-sampled
+and/or co-added images and one should keep in mind that, for large
+images, the pixels are not exactly iso-area. One point of convention
+that might be useful to keep in mind; is that WCS conventions express
+angles in degrees. In the gnomonic projection, offsets in the tangent
+plane are expressed in degrees (defined through angles along great
+circles at the tangent point), so that the metric in the tangent plane
+is ortho-normal), and sidereal angles evaluated on the sky are also
+provided in degrees by the standard implementations. A notable exception
+is the LSST software stack where, by default, the angles are provided
```
Comment (timj) [inline DMTN-036.tex:674]: AST also uses radians.

File: DMTN-036.tex [hunk 12]
```diff
@@ -0,0 +1,808 @@
...
+implements the SIP addition, which consists in applying a 2-d polynomial
+transform to the CCD space coordinates, prior to entering the standard
+WCS chain (affine transform, then de-projection). In practice, the
+SIP ``twisting'' is applied by the LSST software itself (in the
+class \ClName{afw::image::TanWcs}), and the ``standard'' part
+(affine and de-projection, or the reverse transform) are sub-contracted
+to the ``libwcs'' code.
+
+One common complication of the WCS arena is that it was designed in
+the FITS framework convention, itself highly fortran-biased for array
```
Comment (timj) [inline DMTN-036.tex]: "Fortran"

File: DMTN-036.tex [hunk 13]
```diff
@@ -0,0 +1,808 @@
...
+pixel space, that we call $P_{A}$, for the forward transformation.
+By convention, the transform from pixel space to tangent plane then
+reads: 
+\[
+Y_{TP}=L\left(X_{pix}-X_{ref}+P_{A}(X_{pix}-X_{ref})\right)
+\]
+which again does not depend on the frame choice (0-based or 1-based),
+provided $X_{pix}$ and $X_{ref}$ are expressed in the same frame.
+
+In jointcal ,the internal representation of SIP WCS's uses three straight
```
Comment (timj) [inline DMTN-036.tex]: Should "jointcal" be `\textsc{jointcal}` or something?

File: DMTN-036.tex [hunk 14]
```diff
@@ -0,0 +1,808 @@
...
+transformation and the de-projection. Those are just composed to yield
+the actual transform, and the two first ones are generic polynomial
+transformations. We provide routines to translate the TanWcs objects
+into our representation (\RoutineName{ConvertTanWcs}) and back (\RoutineName{GtransfoToTanWcs}).
+In the latter case, we also derive the reverse distortion polynomials,
+which are built if needed in our representation of SIP WCSs.
+
+\section{Notes on meas\_mosaic (from HSC)}
+
+Naoki Yasuda\emph{ }wrote \emph{meas\_mosaic} for HSC processing,
```
Comment (timj) [inline DMTN-036.tex]: That empty `\emph` seems wrong.

**Review by timj — APPROVED** [review]
Looks like a good start. Haven't read through in detail but I have a few fixes that are needed.

**Comment by timj** [comment]
```
commit b30332deaf0335b7c1616dee5a8b22670676e77c
Author: John K. Parejko <parejkoj@fiddlercrab.local>
Date:   Tue Sep 5 11:51:52 2017 -0700
```
Note `.local`.

File: DMTN-036.tex (hunk 1 above)
Comment (parejkoj) [reply DMTN-036.tex]: I have no idea what you mean by that. The `newcommand` blocks here were in the original document from Pierre. What should I use instead?

File: DMTN-036.tex (hunk 2 above)
Comment (parejkoj) [reply DMTN-036.tex]: What does "release" count as? I'll merge to master, but this is by no means "complete".

File: DMTN-036.tex (hunk 1 above)
Comment (timj) [reply DMTN-036.tex]: `\textbf` not `{\bf`

File: DMTN-036.tex (hunk 2 above)
Comment (timj) [reply DMTN-036.tex]: Release probably means "when the JIRA ticket is done". You don't want a tweak to the build system next year to change the document date.

File: DMTN-036.tex (hunk 9 above)
Comment (parejkoj) [reply DMTN-036.tex]: Ah, the one defined in lsst-texmf? It's not described here that I can see: http://lsst-texmf.lsst.io/

There's a whole bunch of macros defined in lsst-texmf, but I don't know what they're each for, though I think I can guess (e.g., `\code{}`, `\class{}` (which are defined in different sections, `\method{}`).

Comment (timj) [reply DMTN-036.tex]: Yes. I haven't worked out how to document them all without duplicating everything.

File: DMTN-036.tex (hunk 4 above)
Comment (parejkoj) [reply DMTN-036.tex:38]: That's an auto-insert by LyX. I'll ask them about adding it to the list of things that can be defined as class-provided in the layout.

File: DMTN-036.tex (hunk 3 above)
Comment (parejkoj) [reply DMTN-036.tex]: Didn't know that ToC was provided by lsst-texmf.

File: DMTN-036.tex (hunk 6 above)
Comment (parejkoj) [reply DMTN-036.tex:63]: What is the best reference for Gaia DR1?

File: DMTN-036.tex (hunk 5 above)
Comment (parejkoj) [reply DMTN-036.tex]: What's the preferred reference for SCAMP?

File: DMTN-036.tex (hunk 6 above)
Comment (timj) [reply DMTN-036.tex:63]: No idea. https://ui.adsabs.harvard.edu/#abs/2016A%26A...595A...2G/abstract ?

File: DMTN-036.tex (hunk 5 above)
Comment (timj) [reply DMTN-036.tex]: https://ui.adsabs.harvard.edu/#abs/2006ASPC..351..112B/abstract

File: DMTN-036.tex (hunk 10 above)
Comment (parejkoj) [reply DMTN-036.tex]: I'm removing that whole section, since there's a `jointcalTask` now.

File: DMTN-036.tex (hunk 13 above)
Comment (parejkoj) [reply DMTN-036.tex]: I don't know. Is there a `\package{}` or equivalent defined in lsst-texmf?

Comment (timj) [reply DMTN-036.tex]: I don't think there is. We could add one. This is my MNRAS style guide persona kicking in.

File: DMTN-036.tex (hunk 1 above)
Comment (parejkoj) [reply DMTN-036.tex]: I've removed both of those commands and just used the `\code{}` and `\method{}` commands from lsst-texmf.

File: DMTN-036.tex (hunk 9 above)
Comment (parejkoj) [reply DMTN-036.tex]: I'm using `code` and `method` now.

File: DMTN-036.tex (hunk 4 above)
Comment (parejkoj) [reply DMTN-036.tex:38]: Upstream request sent. I'll leave this as is for now (it doesn't break the build).

File: DMTN-036.tex (hunk 13 above)
Comment (parejkoj) [reply DMTN-036.tex]: There is `\pack{}`, which I assume means `\package{}` (and which I'd suggest we rename to that...)

File: DMTN-036.tex (hunk 4 above)
Comment (parejkoj) [reply DMTN-036.tex:38]: Upstream response is that this will be a configurable in the layout in LyX 2.3.

File: DMTN-036.tex (hunk 7 above)
Comment (parejkoj) [reply DMTN-036.tex]: I've switched to using formatted references with prettyref, which handles it automatically.

File: DMTN-036.tex (hunk 11 above)
Comment (parejkoj) [reply DMTN-036.tex:674]: I'm not sure about the utility of this whole section, so I'm going to leave it for now.

File: DMTN-036.tex (hunk 13 above)
Comment (parejkoj) [reply DMTN-036.tex]: I've used `\code` for all of these, which does the job.

**Pull request merged**